Business Rules/RITM state change/Related_task_state_update.js

```diff
@@ -0,0 +1,16 @@
+//create business rule on sc_req_item table and add condtion 
+
+(function executeRule(current, previous /*null when async*/ ) {
+
+    var gr = new GlideRecord("sc_task");
+    gr.addQuery('request_item', current.sys_id);
+    gr.orderBy('order');
```

Comment:
You don't need to sort the results if you don't have a clear reason, in this case, you are sorting a list with one record.

Reply:
removed the orderBy query